src/tools/ScenarioMeasurement/Startup/Startup.cs

```diff
@@ -116,7 +120,15 @@ static void checkArg(string arg, string name)
             if (!failed)
             {
                 logger.Log("Parsing..");
-                TraceEventSession.Merge(new[] { kernelTraceFile, userTraceFile }, traceFileName);
+                var files = new System.Collections.Generic.List<string>
+                {
+                    kernelTraceFile
+                };
```

Comment:
nit: you can put this all on one line:
```suggestion
var files = new System.Collections.Generic.List<string> { kernelTraceFile };
```